### DISCUSSION TITLE
[SOVED] Appreciation post

### DISCUSSION THREAD

**Discussion by slalaurette** [martinrotter/rssguard#1016] [General]
Since I decided to phase out my usage of QuiteRSS and adopt RSS Guard I've brought up several issues and as I use the program more and more often, I'm sure I'll come up with some more. So I thought it was time to post this to show my appreciation to the developer instead of just bringing up another issue. 

As I understand, RSS Guard is a one-man operation, and Martin must have a great dedication to fix and improve such a complex app, which is way more than we deserve. ツ He strives to respond to user queries and fix or improve his app. And we get the benefits for free.

I'll see if it's possible for me to donate a couple of bucks (not sure if I'll be able from my country, whose relationship with the US dollar is... complicated) but regardless of the material token of appreciation, I wanted to say it here: Martin, you are the MVP.

**Comment by martinrotter** [comment]
Thanks man.

**Reply by slalaurette** [reply]
BTW I had seen a donation link but I can't find it anymore. Where is it?

**Reply by martinrotter** [reply]
Open your RSS Guard -> main menu -> Help -> Donate

**Reply by martinrotter** [reply]
or in docs

https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#donat

**Reply by slalaurette** [reply]
Done (I think), thank you!

**Reply by martinrotter** [reply]
Thanks a ton  man.
